packages/cli/src/util/__tests__/colorizer.spec.ts

```diff
@@ -0,0 +1,74 @@
+import chalk from 'chalk';
+import {
+  PRE_PARAM_VALUE_TAG,
+  POST_PARAM_VALUE_TAG,
+  attachTagsToParamsValues,
+  transformPathParamsValues
+} from '../colorizer';
+
+describe('colorizer', () => {
+
+  describe('transformPathParamsValues()', () => {
+    it('colorizes tagged values of query params', () => {
+      const path = `/no_auth/pets/findByStatus?status=${PRE_PARAM_VALUE_TAG},sold,pending,${POST_PARAM_VALUE_TAG}`;
```

Comment:
This is a good test but I think it is based on a bug? This status is an enum and should only ever contain one thing, but we're passing in all the things. Functionally it doesnt matter because it is just an arbitrary string, but it makes it look like this sort of thing is intentional.

Reply:
@philsturgeon, it might be a bug. But maybe it was intended @StefanDywersant? cc @XVincentX

Reply:
@philsturgeon I agree that it does not look correct, but it is perfectly correct. See the schema of `status` parameter (https://github.com/stoplightio/prism/blob/master/examples/petstore.oas3.json#L112). It is array of enums.

Reply:
@StefanDywersant, @philsturgeon pointed out that "This status is an enum and should only ever contain one thing, but we're passing in all the things.", so it is an enum, but it still should only produce one value for the generated path to be valid (?)

Reply:
Ha! Ok it's correct! Just looks super odd :D Thanks for clearing that up.